templates/postgresql_database.yml.erb

```diff
@@ -4,9 +4,19 @@ development: &default
   encoding: utf8
   host: localhost
   min_messages: warning
-  pool: 2
+  pool: <%%= ENV.fetch("DB_POOL", 2) %>
+  reaping_frequency: <%%= ENV.fetch("DB_REAPING_FREQUENCY", 10) %>
   timeout: 5000
 
 test:
   <<: *default
   database: <%= app_name %>_test
+
+production: &deploy
+  encoding: utf8
+  min_messages: warning
+  pool: <%%= [ENV.fetch("MAX_THREADS", 2), ENV.fetch("DB_POOL", 2)].max %>
```

Comment:
The DB_POOL needs to be greater than or equal to the MAX_THREADS so defaulting it to 2 will lead to connection issues. On FormKeep we had this happen and since rack-timeout is on the main thread it killed the connection and left a prepared statement in the pool causing tons of errors. I suggest upping this to 5 to be in sync with the default on MAX_THREADS